data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -51,21 +51,21 @@
     private final AuditMessageService auditMessageService;
 
     /**
-     * Constructor for the DataController
+     * Constructor for the DataController.
      *
-     * @param auditableDataService service for providing auditable data for the request
-     * @param auditMessageService  service for sending audit messages
+     * @param auditableDataService service for providing data retrieval service for the request
+     * @param auditMessageService  service for sending audit success and error messages to the Audit Service
      */
     public DataController(final AuditableDataService auditableDataService, final AuditMessageService auditMessageService) {
         this.auditableDataService = auditableDataService;
         this.auditMessageService = auditMessageService;
     }
 
     /**
-     * REST endpoint to read a resource and return a streaming response.
+     * REST endpoint to read a resource request and return a streaming response.
      *
-     * @param dataRequest the request to read the data from a leafResource
-     * @return a stream of bytes representing the contents of the resource
+     * @param dataRequest the requested data
+     * @return a stream of holding the contents of the response
```

Comment:
This could do with being a bit more descriptive. What is the contents of the response?

Reply:
re-worded to be in-line with the method description.